src/main/java/blackjack/domain/card/CardBundle.java

```diff
@@ -0,0 +1,42 @@
+package blackjack.domain.card;
+
+import blackjack.domain.score.ScoreCalculator;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class CardBundle {
+
+    private static final int BLACK_JACK_SCORE = 21;
+
+    private final List<Card> cards;
+
+    private CardBundle() {
+        this.cards = new ArrayList<>();
+    }
+
+    public static CardBundle emptyBundle() {
+        return new CardBundle();
+    }
+
+    public void addCard(Card card) {
+        cards.add(card);
+    }
+
+    public int calculateScore() {
+        return ScoreCalculator.findByCards(cards)
+            .calculateScore(cards);
+    }
+
+    public List<Card> getCards() {
+        return Collections.unmodifiableList(cards);
```

Comment:
```suggestion
        return new ArrayList<>cards);
```
`Collections.unmodifiableList`는 방어적 복사가 되지 않습니다.

참고 자료 
- https://tecoble.techcourse.co.kr/post/2021-04-26-defensive-copy-vs-unmodifiable/